sample-code/src/server.ts

```diff
@@ -249,11 +250,18 @@ app.get('/orchestration/:sampleCase', async (req, res) => {
       template: orchestrationTemplating,
       inputFiltering: orchestrationInputFiltering,
       outputFiltering: orchestrationOutputFiltering,
-      requestConfig: orchestrationRequestConfig
+      requestConfig: orchestrationRequestConfig,
+      fromJSON: orchestrationFromJSON
     }[sampleCase] || orchestrationChatCompletion;
 
   try {
-    const result = (await testCase()) as OrchestrationResponse;
+    const result =
+      sampleCase === 'fromJSON'
+        ? // File path should be relative to the root. For this example, `sample-code` is the root.
+          ((await testCase(
+            './src/ModelOrchConfig.json'
```

Comment:
[req] style guide for filename please https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#filenames-and-file-paths